src/stores/SchemaStore.ts

```diff
@@ -32,29 +34,60 @@ export class SchemaStore {
 	history = new HistoryStore(this);
 
 	requestsStore: RequestsStore;
-	
+
 	selectedDictionaryStore: SelectedDictionaryStore;
-	
+
 	boxUpdater: BoxUpdater;
 
 	dictionaryLinksStore: DictionaryLinksStore;
 
+	subscriptionStore: SubscriptionStore;
+
+	schemaSettings: SchemaSettings | null;
```

Comment:
schemaSettings should be observable. Otherwise, your reaction in SubscriptionStore won't work